README.md

```diff
@@ -36,6 +36,26 @@ more easily call Google's gRPC and REST APIs. Many of these libraries are
 generated using [toolkit](https://github.com/googleapis/toolkit) and
 [artman](https://github.com/googleapis/artman).
 
+### Google API Extensions (GAX)
+
+The [googleapis](https://github.com/googleapis) organization also hosts
+repositories that contain common code used by Google's generated and handwritten
+client libraries. Details vary by language, but most provide support
+for generally-needed features like paging and retry. Currently there is one such
+library for each of the main languages that we support:
+[gax-dotnet](https://github.com/googleapis/gax-dotnet),
+[gax-go](https://github.com/googleapis/gax-go),
+[gax-java](https://github.com/googleapis/gax-java),
+[gax-nodejs](https://github.com/googleapis/gax-nodejs),
+[gax-php](https://github.com/googleapis/gax-php),
+[gax-python](https://github.com/googleapis/gax-python), and
```

Comment:
This one is ~deprecated~ no longer used in favor of api-core (which is part of [google-cloud-python](https://github.com/GoogleCloudPlatform/google-cloud-python.git) for the moment). Jon needed a different name in order to facilitate a transition while not migrating the world at once.

I will sanity check that gax-python is no-really-dead and then archive the repo.